db/revision_cache.go

```diff
@@ -127,7 +115,7 @@ func (rc *RevisionCache) GetActive(docid string, context *DatabaseContext) (docR
 
 	// Retrieve from or add to rev cache
 	value := rc.getValue(docid, bucketDoc.CurrentRev, true)
-	docRev, statEvent, err := value.loadForDoc(bucketDoc, context)
+	docRev, statEvent, err := value.loadForDoc(bucketDoc, context, BodyShallowCopy)
```

Comment:
I think this might have been my fault from another PR, but I think the function signature:

```
func (value *revCacheValue) load(loaderFunc RevisionCacheLoaderFunc, copyType BodyCopyType) (DocumentRevision, bool, error) {

```

should be changed at some point to have a named return type.  I think just having `bool` is a bit unclear, and puts the burden on the person reading the code to look at internal variable names or calling code to figure out the nature of the return val.